specification/vmware/Microsoft.AVS.Management/workloadNetworks-dnsServices.tsp

```diff
@@ -43,18 +42,13 @@ interface WorkloadNetworkDnsServices {
     WorkloadNetworkDnsService
   >;
 
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-delete-operation-response-codes" "Can not change existing response codes."
   #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-operation" "DnsServiceIdParameter should be last, but it is breaking to correct it."
   #suppress "@azure-tools/typespec-azure-core/no-openapi" "Can not change existing operationId."
   @operationId("WorkloadNetworks_DeleteDnsService")
-  // delete is ArmResourceDeleteAsync<WorkloadNetworkDnsService>;
   @route("/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AVS/privateClouds/{privateCloudName}/workloadNetworks/default/dnsServices/{dnsServiceId}")
   @doc("Delete a {name}", WorkloadNetworkDnsService)
-  @delete
-  @extension("x-ms-long-running-operation", true)
-  @extension(
-    "x-ms-long-running-operation-options",
-    #{ `final-state-via`: "location" }
-  )
+  @armResourceDelete(WorkloadNetworkDnsService)
```

Comment:
nit, delete as LRO seems to work as the response is 
```
    | ArmDeletedResponse
    | ArmDeleteAcceptedLroResponse
    | ArmDeletedNoContentResponse
    | ErrorResponse;
```

Reply:
@weidongxu-microsoft , I'm not sure what you mean.

Reply:
As this is a nit, I'm going to merge the PR without this change, but I would still like to know in case it can improve our spec.